packages/components/src/toggle-group-control/stories/index.js

```diff
@@ -151,20 +157,19 @@ export const WithIcons = ( props ) => {
 			<ToggleGroupControlOptionIcon
 				value="uppercase"
 				icon={ formatUppercase }
-				showTooltip={ true }
-				aria-label="Uppercase"
+				label="Uppercase"
 			/>
 			<ToggleGroupControlOptionIcon
 				value="lowercase"
 				icon={ formatLowercase }
-				showTooltip={ true }
-				aria-label="Lowercase"
+				label="Lowercase"
```

Comment:
We should update also the [unit tests](https://github.com/WordPress/gutenberg/blob/98a478f05bbd645e15d2b4b23525c7a00e0ba431/packages/components/src/toggle-group-control/test/index.js#L59-L70) and probably add a changelog entry about adding the `label` prop to the component and always showing a tooltip?

Reply:
Great idea. I've also converted the test file to a `tsx`. (no other errors! yay)